entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnode.java

```diff
@@ -43,13 +67,13 @@ default String getName() {
 
     MergedXnode setXnodeQ2(double xNodeQ2);
 
-    String getLine1Name();
+    float getB2dp();
```

Comment:
Out of curiosity, is there a reason you are using `float` (instead of `double` for example)?

Reply:
To be honest i don't know, I just did like rdp and xdp

Reply:
No worry! It was just a question, it seems indeed logical to use the same thing as `rdp` and `xdp`